app/forms/wpcc/object_error.rb

```diff
@@ -0,0 +1,16 @@
+module Wpcc
+  class ObjectError < Dough::Forms::ObjectError
+    MESSAGES = [
+      I18n.t('activemodel.errors.models.wpcc/your_details_form.attributes.age.greater_than_or_equal_to'),
+      I18n.t('activemodel.errors.models.wpcc/your_details_form.attributes.age.less_than')
+    ].freeze
+
+    def full_message
+      if field_name == :age && message.in?(MESSAGES)
+        message
+      else
+        super
+      end
+    end
+  end
+end
```

Comment:
@margOnline I think you need to be aware of the mechanisms of the two files above (app/forms/wpcc/object_error.rb and app/forms/wpcc/builder.rb).